File: Facturacion/Models/FacturaLista.cs
using System;
using System.Collections.Generic;

namespace Facturacion.Models
{
    public partial class FacturaLista
    {
        public int IdFactura { get; set; }
        public int IdCliente { get; set; }
        public string Vendedor { get; set; }
        public decimal Neto { get; set; }
        public decimal Iva { get; set; }
        public decimal PrecioTotal { get; set; }
        public int Numero { get; set; }

        public Clientes IdClienteNavigation { get; set; }
        public CodigoFactura IdFacturaNavigation { get; set; }
    }
}

File: src/app/pages/Models/facturaLista.ts
export class FacturaLista {

  constructor(
    public  IdFactura:number = 0,
    public  IdCliente:number = 0,
    public  IdProductos:number = 0,
    public  Cantidad: number = 0,
    public  Vendedor:string ='',
    public  Neto: number = 0,
    public  Iva: number = 0,
    public  PrecioTotal: number = 0,
    public  Numero: number = 0
  ) { }

}
